src/man.rs

```diff
@@ -349,15 +356,32 @@ fn env(page: Roff, environment: &[Env]) -> Roff {
 ///
 ///        2      Optional error
 /// ```
-fn exit_status(page: Roff) -> Roff {
-  page.section(
-    "EXIT STATUS",
-    &[
+fn exit_status(page: Roff, exit_statuses: &[ExitStatus]) -> Roff {
+  if exit_statuses.is_empty() {
+    return page;
+  }
+  let arr = if exit_statuses
+    .iter()
+    .any(|status| status.use_default_instead)
+  {
```

Comment:
It took me a bit to read this statement. Perhaps we could break this up a bit so it's easier to read?